src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/admin_user/create.yaml

```diff
@@ -11,6 +11,8 @@ twig_hooks:
                 template: '@SyliusAdmin/admin_user/form/sections.html.twig'
 
         'sylius_admin.admin_user.create.content.form.sections':
+            general:
```

Comment:
in BootstrapAdminUi, have have some default Twig hooks to use form_widget(form) and not having an empty page which will be very surprising for DX.